evaluation/agent_bench/README.md

```diff
@@ -18,7 +18,7 @@ cache_dir = "/path/to/cache"
 workspace_base = "/path/to/workspace"
 workspace_mount_path = "/path/to/workspace"
 
-sandbox_container_image = "ghcr.io/opendevin/sandbox:latest"
+sandbox_container_image = "ghcr.io/opendevin/sandbox:main"
```

Comment:
We should just remove this. 2 reasons:

1) if `sandbox_container_image` is not specified, it defaults to `main`.
2) with #2491, now we build image on the fly, so this is not used anymore.


```suggestion
```